string departure_station
---
TrainData[] train_list